CMakeLists.txt

```diff
@@ -204,7 +220,11 @@ IF(BOOST_STATIC)
     set(Boost_USE_STATIC_LIBS   ON)
 endif()
 
-find_package(OpenSceneGraph 3.2.0 REQUIRED osgDB osgViewer osgText osgGA osgAnimation osgParticle osgQt osgUtil osgFX)
+if (NOT ANDROID)
+    find_package(OpenSceneGraph 3.2.0 REQUIRED osgDB osgViewer osgText osgGA osgAnimation osgParticle osgQt osgUtil osgFX)
```

Comment:
<pre>
set (OSG_COMPONENTS osgDB osgViewer osgText osgGA osgAnimation osgParticle osgQt osgUtil osgFX)
if (NOT ANDROID)
   set (OSG_COMPONENTS ${OSG_COMPONENTS} osgQt)
endif()

find_package(OpenSceneGraph 3.2.0 REQUIRED ${OSG_COMPONENTS})
</pre>